internal/criocli/config.go

```diff
@@ -60,10 +61,7 @@ it later with **--config**. Global options will modify the output.`,
 		}
 
 		if c.IsSet("migrate-defaults") {
-			logrus.Infof("Migrating config from %s", from)
-			if err := migrate.Config(conf, from); err != nil {
-				return fmt.Errorf("migrate config: %w", err)
-			}
+			return errors.New("config migration has been deprecated and will be removed in v1.29.0")
```

Comment:
I'm a bit on the fence about what "deprecation" means here. Usually I'd expect that it still works as intended but will be removed in 1.29 (aka does not work any more).